azure-pipelines.yaml

```diff
@@ -112,3 +121,17 @@ stages:
               pr_number: $(pull_request_number)
               merge_method: 'merge'
             displayName: Merge PR# $(System.PullRequest.PullRequestNumber)
+
+  - stage: PoliCheck
+    dependsOn: ['Windows', 'Mac']
+    displayName: PoliCheck
+    jobs:
+    # Check - "PoliCheck "
+    - job: run_poli_check
```

Comment:
for consistency, use run_polichack_compliance with a display name of "Policheck Compliance"

Reply:
Done.